.github/workflows/release.yml

```diff
@@ -3,16 +3,90 @@ name: Release
 
 on:
   pull_request:
-    types: closed
-
+    types: [ closed ]
```

Comment:
Looks like you're turning this into an array, does it need to be now or is it just preferred?